### PR TITLE
Rd 1610 ansible step

### PR DESCRIPTION
**Pull request by EarthmanT**

### CODE REVIEW

File: cloudify_ansible/utils.py [hunk 1]
```diff
@@ -597,3 +626,358 @@ def is_connected_to_internet():
...
+        file_name=playbook_path,
+        variable_manager=variable_manager,
+        loader=loader)
+
+
+def get_available_steps(playbook_path):
+    return get_tags_from_playbook(get_playbook_from_path(playbook_path))
+
+
+def get_our_tags(all_tags, op_number, max_ops):
```
Comment (AdarShaked) [inline cloudify_ansible/utils.py:927]: Can you give an example of the invocation of this function? cant understand 100% its purpose

File: plugin.yaml [hunk 2]
```diff
@@ -137,6 +138,18 @@ dsl_definitions:
         A boolean value, `true` or `false` whether
         to assume the user privileges.
       default: false
+    tags:
+      type: list
+      description: >
+        A list of tags, in order of desired execution. If these tags are provided, they will be called, and auto_tags will be ignored.
+      default: []
+    auto_tags:
```
Comment (AdarShaked) [inline plugin.yaml:146]: why actually we need this auto_tags? if the user wants to run the tasks that have tags so he will specify it, if not specified so we will run the playbook as before.

File: cloudify_ansible/__init__.py [hunk 3]
```diff
@@ -48,6 +48,17 @@ def wrapper(group_name=None,
     return wrapper
 
 
+def set_ansible_env_vars(ansible_env_vars=None):
+    ansible_env_vars = ansible_env_vars or {}
+    if constants.OPTION_HOST_CHECKING not in ansible_env_vars:
```
Comment (AdarShaked) [inline cloudify_ansible/__init__.py:53]: Instead of may ifs you can use .get(constants.OPTION_HOST_CHECKING,False)

File: plugin.yaml (hunk 2 above)
Comment (EarthmanT) [reply plugin.yaml:146]: Because users prefer to be lazy.

File: cloudify_ansible/__init__.py (hunk 3 above)
Comment (EarthmanT) [reply cloudify_ansible/__init__.py:53]: Generally, in Python style, if we are not using a value, we don't use `get`.

**Pull request merged**